kinta-cli/src/main/kotlin/com/dailymotion/kinta/command/Init.kt

```diff
@@ -24,8 +25,12 @@ object Init : CliktCommand(name = "init", help = "Initializes kinta in a project
         copyResource("com.dailymotion.kinta.Workflows", "kintaSrc/src/main/resources/META-INF/services/com.dailymotion.kinta.Workflows")
         copyResource("build.gradle.kts", "kintaSrc/build.gradle.kts")
         copyResource("gitignore", "kintaSrc/.gitignore")
+        copyResource("kinta.properties", "kintaSrc/kinta.properties")
 
+        //Assemble kintaSrc
         Gradle(File("kintaSrc")).executeTask("assemble")
+        //Add kintaSrc to Git
+        GitIntegration.add(File("kintaSrc").path)
```

Comment:
Maybe write a message that `kintaSrc` has been added ? 

"This project is now setup to use kinta. You can define your workflows in kintaSrc/src/main/.... Take a look around. If you like what you see, commit your kintaSrc directory. It is meant to be in source control."

Reply:
done